server/app/services/docker/service_creator.rb

```diff
@@ -131,7 +131,7 @@ def build_labels
         labels['io.kontena.load_balancer.internal_port'] = internal_port
         labels['io.kontena.load_balancer.mode'] = mode
       end
-      if grid_service.health_check
+      if grid_service.health_check && grid_service.health_check.protocol
```

Comment:
Is only protocol only required? Should we check that all health check options are in place?